### PR TITLE
Added attack animations.

### PR DESCRIPTION
**Pull request by ComicIronic**
Lot of work to add this one in!

After months of hard coding, I've finally added attack animations to the codebase in a state where everyone is happy!

### Actually

Added attack animations to the changelog.

### CODE REVIEW

File: html/changelogs/revert-attack-anims.yml [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+#   rscadd (general adding of nice things)
+#   imageadd
+#   imagedel
+#   spellcheck (typo fixes)
+#   experiment
+#   tgs (TG-ported fixes?)
+#################################
+
+# Your name.
+author: ComicIronic
```
Comment (PJB3005) [inline html/changelogs/revert-attack-anims.yml]: Actually this should be under Cluster's name.

**Comment by salamandooo** [comment]
Cool

**Comment by BobdaBiscuit** [comment]
\>didn't secretly add TRAINS in this pr

Shit tier tbh.

**Comment by SonixApache** [comment]
funny, i actually thought about this but figured it'd be kind of a bitchy thing to do

@gbasood dude what

**Comment by Intigracy** [comment]
This is just a changelog anyway.

**Comment by gbasood** [comment]
that's
the joke

**Pull request merged**